api/src/main/resources/underlays/omop_test.yaml

```diff
@@ -0,0 +1,437 @@
+# An initial OMOP underlay schema for development.
+
+name: omop_test
+entities:
+  - name: person
+    attributes:
+      - name: person_id
+        dataType: INT64
+      - name: gender_concept_id
+        dataType: INT64
+      - name: gender
+        dataType: STRING
+      - name: race_concept_id
+        dataType: INT64
+      - name: race
+        dataType: STRING
+      - name: ethnicity_concept_id
+        dataType: INT64
+      - name: ethnicity
+        dataType: STRING
+      - name: sex_at_birth_concept_id
+        dataType: INT64
+      - name: sex_at_birth
+        dataType: STRING
+  - name: condition_occurrence
+    attributes:
+      - name: condition_occurrence_id
+        dataType: INT64
+      - name: person_id
+        dataType: INT64
+      - name: condition_concept_id
+        dataType: INT64
+      - name: condition_name
+        dataType: STRING
+      - name: condition_standard
+        dataType: STRING
+      - name: condition_concept_code
+        dataType: INT64
+  - name: concept
+    attributes:
+      - name: concept_id
+        dataType: INT64
+      - name: concept_name
+        dataType: STRING
+      - name: domain_id
+        dataType: STRING
+      - name: vocabulary_id
+        dataType: INT64
+      - name: vocabulary_name
+        dataType: STRING
+      - name: standard_concept
+        dataType: STRING
+      - name: concept_code
+        dataType: INT64
+relationships:
+  - name: person_condition_occurrence
+    entity1: person
+    entity2: condition_occurrence
+  - name: person_gender_concept_id
+    entity1: concept
```

Comment:
Hmm I'm surprised by the inversion of order on this, as compared to the first entry. Is this meaningful? If not, maybe just a convention to mirror the order in the name could make sense

Reply:
right now it matters for the related `relationshipMapping` where the entity1 is expected to be the entity twith the table as the primary key. Did some renaming.

Reply:
Got it. may be worth renaming from entity1 and entity2 if these have semantics (probably not in this PR), maybe something like entity and foreignEntity?

Reply:
SG, will do in a follow up PR